### PR TITLE
Instructor onboarding

### LINKED ISSUE #4: [feat] Instructor register/onboarding
User Story: 
As a Instructor-User, I want to set and save my infos and skills, to be seen by other users.

AC: 
- [x] Form Input element exists with needed information for card
- [x] Submitted Information is saved locally with zustand
- [x] User gets a confirmation of upload
- [x] Card can be rendered out of stored information 

Complexity Index: M | L

### PR DESCRIPTION
**Pull request by haskamp**
## Motivation
Create a Form to get information form instructor
Information is stored in Zustand 
Card can get rendered out of stored information
Wizzard excluded due to size 

## Issues closed
closes #4, opens #18 image upload, #20 Form Wizzard and #19 Form Design Feature

Screenshot 2022-02-05 at 14 00 51
s

### CODE REVIEW

File: src/ions/hooks/useLog.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "react";
+
+const useLog = (key, value) => {
+	useEffect(() => {
+		console.log(key, value);
```
Comment (konrascher3) [inline src/ions/hooks/useLog.jsx:5]: Remove unused console.log

File: src/ions/templates/form/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	{ label: "Sankt-Peter-Ording" },
+];
+export const license = [{ label: "IKO" }, { label: "VDWS" }, { label: "VDS" }];
+
+const formTemplateCheckbox = {
+	groups: [
+		{
+			id: "availabilities",
+			legend: "Your availabilities",
+			helperText: "When are free to teach?",
```
Comment (konrascher3) [inline src/ions/templates/form/index.js:15]: Missing word

File: src/molecules/alert-submit/index.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Alert, AlertTitle, Snackbar } from "@mui/material";
+import React from "react";
+import useStore from "../../ions/store";
```
Comment (konrascher3) [inline src/molecules/alert-submit/index.jsx:3]: use absolute imports e.g. "/src/bla/bla.jsx"

File: src/molecules/card-availabilities/index.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Box, Button, ButtonGroup } from "@mui/material";
+import React from "react";
+import useStore from "../../ions/store";
```
Comment (konrascher3) [inline src/molecules/card-availabilities/index.jsx:3]: Absolute imports

File: src/molecules/card-extended-content/index.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import { CardContent, Collapse, Divider, Stack, Typography } from "@mui/material";
+import React from "react";
+import useStore from "../../ions/store";
+
+import CardAvailabilities from "../card-availabilities";
+import CardLanguages from "../card-languages";
+import CardServices from "../card-services";
+
```
Comment (konrascher3) [inline src/molecules/card-extended-content/index.jsx:8]: Imports

File: src/molecules/card-footer/index.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { Button, CardActions, IconButton, Tooltip, Typography } from "@mui/material";
+import { ExpandMore, Message } from "@mui/icons-material";
+import React from "react";
+import useStore from "../../ions/store";
```
Comment (konrascher3) [inline src/molecules/card-footer/index.jsx:4]: Imports

File: src/molecules/card-services/index.jsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import useStore from "../../ions/store";
```
Comment (konrascher3) [inline src/molecules/card-services/index.jsx:1]: Imports

File: src/organisms/card/index.jsx [hunk 8]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Card } from "@mui/material";
+import useStore from "../../ions/store";
```
Comment (konrascher3) [inline src/organisms/card/index.jsx:3]: Imports

File: src/organisms/form-lesson/index.jsx [hunk 9]
```diff
@@ -0,0 +1,111 @@
+import {
+	Autocomplete,
+	FormControlLabel,
+	FormGroup,
+	FormHelperText,
+	FormLabel,
+	Stack,
+	TextField,
+} from "@mui/material";
+import formTemplateCheckbox, { license, locations } from "../../ions/templates/form";
```
Comment (konrascher3) [inline src/organisms/form-lesson/index.jsx:10]: imports

File: src/organisms/form/index.jsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Button } from "@mui/material";
+import useStore from "../../ions/store";
+
+import formTemplateCheckbox from "../../ions/templates/form";
+import FormBasic from "../form-basic";
+import FormLesson from "../form-lesson";
+import FormAbout from "../form-about";
+import useLog from "../../ions/hooks/useLog";
+import AlertSubmit from "../../molecules/alert-submit";
```
Comment (konrascher3) [inline src/organisms/form/index.jsx:10]: Imports

**Comment by konrascher3** [comment]
Sonst passts

File: src/ions/hooks/useLog.jsx (hunk 1 above)
Comment (haskamp) [reply src/ions/hooks/useLog.jsx:5]: is an utility func to log stuff

**Pull request merged**